src/main/resources/assets/mooblooms/lang/de_de.json

```diff
@@ -0,0 +1,168 @@
+{
+	"entity.mooblooms.dandelion_Blumenkuh": "Löwenzahnblumenkuh",
+	"entity.mooblooms.poppy_Blumenkuh": "Mohnblumenkuh",
+	"entity.mooblooms.blue_orchid_Blumenkuh": "Blaue Orchideenblumenkuh",
+	"entity.mooblooms.allium_Blumenkuh": "Zierlauchblumenkuh",
+	"entity.mooblooms.oxeye_daisy_Blumenkuh": "Margeritenblumenkuh",
+	"entity.mooblooms.cornflower_Blumenkuh": "Kornblumenblumenkuh",
```

Comment:
Kornblumenkuh

Reply:
I've named it that way, because the flower is called 'Kornblume' in German.